tf2_eigen/test/tf2_eigen-test.cpp

```diff
@@ -28,13 +28,9 @@
 
 
 #include <tf2_eigen/tf2_eigen.h>
-#include <ros/ros.h>
 #include <gtest/gtest.h>
 #include <tf2/convert.h>
 
-static const double EPS = 1e-3;
-
-
 TEST(TfEigen, ConvertVector)
```

Comment:
Actually, it turns out that only the EPS removal can go upstream; the PR is here: https://github.com/ros/geometry2/pull/215 .  The ros.h header is needed upstream to get access to the ros::Time symbol, which is still used to construct Stamped objects in ROS1.